stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.3.3
+version: 2.3.4
```

Comment:
Following [semver versioning](https://semver.org/), as you are introducing a new feature to the chart, you should update the minor version and not the patch one. Please update to `2.4.0`

Reply:
Done.